demo/src/right-resizable-box.js

```diff
@@ -66,27 +68,26 @@ const RightResizableBox = (props) => {
     };
 
     return (
-        <ResizableBox
+        <ResizableBoxSx
             style={{ display: props.hide ? 'none' : undefined }}
```

Comment:
why isn't this in the sx ?

Reply:
I migrated makeStyle/useStyle to emotion but left everything else as is. Can be refactored in a separate PR